src/Files.App/Strings/en-US/Resources.resw

```diff
@@ -2856,4 +2856,7 @@
   <data name="BinEmptyingFailed" xml:space="preserve">
     <value>Failed to empty recycle bin</value>
   </data>
+  <data name="ArchivePasswordDialogDescription" xml:space="preserve">
+    <value>Insert archive password</value>
```

Comment:
```suggestion
  <data name="ArchivePassword" xml:space="preserve">
    <value>Archive password</value>
```

We should probably add this header above the text box

Reply:
What if I set it as the placeholder for the text box? It would be much cleaner than the header

Reply:
It's usually recommended to have a header as placeholders are only good as long as the text box is empty.